e2e/support/helpers/e2e-setup-helpers.js

```diff
@@ -2,6 +2,22 @@ export function snapshot(name) {
   cy.request("POST", `/api/testing/snapshot/${name}`);
 }
 
+/**
```

Comment:
I opted for JSDoc, because if I convert this file to TypeScript now, there could be conflicts.